lib/data/network/upload_file/file_info.dart

```diff
@@ -0,0 +1,40 @@
+
+import 'dart:convert';
+
+import 'package:equatable/equatable.dart';
+import 'package:flutter/foundation.dart';
+import 'package:matrix/matrix.dart';
+import 'package:mime/mime.dart';
+
+class FileInfo with EquatableMixin {
+  final String fileName;
+  final String filePath;
+  final int fileSize;
+  final Stream<List<int>>? readStream;
```

Comment:
Sometimes, developer want to pass directly Stream to instead of file, for example when pick files, it also have choices to return as Stream. 
<img width="457" alt="Screenshot 2023-07-21 at 09 26 24" src="https://github.com/linagora/twake-on-matrix/assets/43041967/01d9b130-e300-45f6-b602-40139afc93f7">